.github/workflows/iam-handle.yml

```diff
@@ -0,0 +1,58 @@
+name: 'aod-iam-handle'
+on:
+  workflow_call:
+    inputs:
+      workload_identity_provider:
+        description: 'The full identifier of the Workload Identity Provider, including the project number, pool name, and provider name.'
+        type: 'string'
+        required: true
+      service_account:
+        description: 'Email address or unique identifier of the Google Cloud service account for which to generate credentials.'
+        type: 'string'
+        required: true
+      path:
```

Comment:
This should be a fixed value? Do we need it to be an input?

Reply:
Ok, we can require the path to be ${{ github.workspace }}/iam.yaml

Reply:
Another option is to make it an input with default, not a big deal at this moment, it is going to be a trivial update if required later.